src/LogModel/Microsoft.VisualStudio.ProjectSystemTools.Shared.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\import\VisualStudio.props" />
+  
+  <PropertyGroup>
+    <RootNamespace>Microsoft.VisualStudio.ProjectSystem.LogModel</RootNamespace>
+    <AssemblyName>Microsoft.VisualStudio.ProjectSystem.LogModel</AssemblyName>
+    <IsProductComponent>false</IsProductComponent>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <None Remove="PublicAPI.Shipped.txt" />
+    <None Remove="PublicAPI.Unshipped.txt" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <AdditionalFiles Include="PublicAPI.Shipped.txt" />
+    <AdditionalFiles Include="PublicAPI.Unshipped.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="Resources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
I don't see a Resources.Designer.cs, is this necessary?

Reply:
Shouldn't be necessary